composer.json

```diff
@@ -17,16 +17,16 @@
         "laminas/laminas-router": "^3.4",
         "laminas/laminas-math": "^3.3",
         "laminas/laminas-hydrator": "^4.2",
-        "laminas/laminas-dependency-plugin": "^2.1",
+        "laminas/laminas-dependency-plugin": "^2.6.0",
         "laminas/laminas-i18n": "^2.9",
         "laminas/laminas-cache-storage-adapter-apcu": "^2.0"
     },
     "require-dev": {
         "captainhook/captainhook": "^5.16",
         "captainhook/plugin-composer": "^5.3",
-        "dvsa/mot-cpms-common": "^1.0.0",
-        "dvsa/mot-cpms-payment-test": "^1.0.0",
-        "laminas/laminas-component-installer": "^2.5",
+        "dvsa/mot-cpms-common": "dev-feature-bl-17035-php-82-update",
+        "dvsa/mot-cpms-payment-test": "dev-feature-bl-17037-php-82-update",
```

Comment:
Create a tag in mot-cpms-common and mot-cpms-payment-test and use it here. You can create a tag before merging to master, and after merging you can correct it if needed.

Reply:
As discussed with @aaronlong , we will create tags once the swimlane is ready